### ISSUE TITLE
Private extension list not getting pulled from repository

### ISSUE THREAD

**Issue by amey2688** [joelspadin-garmin/vscode-private-extension-manager#73]
Hi have confgured my jfrog repository endpoints in extensions.private.json file as specified in readme. it is not still able to fetch deployed packages and just keeps running (see attached screenhot). is there any way to check what is happening in background?
![image](https://user-images.githubusercontent.com/33552841/185108112-de7bdd19-4580-4b36-b293-c0106e0fb9b5.png)
 it has been loading like this since hours . 
any suggestions how to fix this ? is it working for anyone else?

**Comment by amey2688** [comment]
this is how my extensions.private.json looks like

"registries": [
		{"name": "My Private Registry",
		"registry": "https://artifactory.xxx.net/artifactory/api/npm/xxx/",
		"enablePagination":false}
	],
	// List of private extensions which should be recommended for users of this workspace.
	"recommendations": [
	]

**Comment by riddlepl** [comment]
I face the same problem. My registry is hosted in jfrog as well. I've checked the results returned by the server and its looks ok, there is a package present. @amey2688 have you find a solution for that?

Update: ok I've used debugger to figure it out. In my case package.yml for npm package uploaded to the npm registry was lacking `engine` entry and as such it's silently discarded by the `Private extension manager`.
